backend/src/panels/YeomanUIPanel.ts

```diff
@@ -34,7 +34,7 @@ export class YeomanUIPanel extends AbstractWebviewPanel {
 		const filter = GeneratorFilter.create(_.get(uiOptions, "filter"));
 
 		const rpc = new RpcExtension(this.webViewPanel.webview);
-		const vscodeYouiEvents: YouiEvents = new VSCodeYouiEvents(rpc, this.webViewPanel, filter, this.messages);
+		const vscodeYouiEvents: YouiEvents = new VSCodeYouiEvents(rpc, this.webViewPanel, filter, this.messages, this.output);
 		this.output.setChannelName(`${YeomanUIPanel.YEOMAN_UI}.${this.messages.channel_name}`);
```

Comment:
this line should be bewfore new VSCodeYouiEvents

Reply:
done